src/components/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
```

Comment:
This is not a test. 

Use `expect`  and enzyme's `shallow` to do something like this - 

```js
it("always renders a div", () => {
  const divs = renderScreen().find("div");
  expect(divs.length).toBeGreaterThan(0);
});
```

Reply:
On it.